main.c

```diff
@@ -392,12 +393,15 @@ void update_info(void)
 	}
 }
 
-int ptr_third_x(void)
+int nav_button(void)
 {
-	int x, y;
+	int x, y, nw;
 
 	win_cursor_pos(&win, &x, &y);
-	return MAX(0, MIN(2, (x / (win.w * 0.33))));
+	nw = (NAV_WIDTH <= 0 ? 0.33 : NAV_WIDTH) * (NAV_WIDTH >= 1 ? 1 : win.w);
```

Comment:
navigation_width of zero should disable navigation and if you make the variable unsigned you don't have to worry about negatives. 

I also don't think you we need to care about values greater than 1. Of course this is very subjective, but using absolute pixels doesn't make much since at compile time given the window can very greatly at round time. And think the code could be simpler here without that consideration
```
nw = NAV_WIDTH * win.w;
```

Reply:
> navigation_width of zero should disable navigation

That's a good idea, I thought it could be disabled somewhere but no,  so it's nice to disable it with value of 0.

> if you make the variable unsigned you don't have to worry about negatives.

> I also don't think you we need to care about values greater than 1. Of course this is very subjective, but using absolute pixels doesn't make much since at compile time given the window can very greatly at round time.

But it's currently a floating number ... Since I am considering two cases, using unsigned integers won't cover both. A workaround is to make it int, and 0 disables it, positive means percentage of window width, and negative means absolute width (I find it better).

As for the absolute width, I was thinking about buttons in GUI programs, they don't change their sizes during window resizing. And, this ensures there always be certain area for navigation action, instead of being too small to click if the window is small, too. And indeed, it's a personal preference, I think it's not bad to provide the choice.

Reply:
Fair point. Slipped my mind that unsigned floats aren't a thing. The 3 cases for int work, but that would leave all numbers greater than 100 useless which may be fine

I don't think what you have is bad. Just wounder if we are trying to cover too many cases. I don't strongly oppose having a support for abs width... feel free to resolve the this thread once it can be disabled

Reply:
Done. I have to return specifically for NAV_WIDTH == 0, since I found on some WMs like dwm, cursor on window border seems to have x coordinate outside (0, win.w), so navigation still works on window border.